packages/react-core-notifications/.estlintrc.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  extends: ['amplify-ui/react'],
```

Comment:
Should we extend `amplify-ui/react-core` instead (which extends `amplify-ui/react`)?